config/config.go

```diff
@@ -94,6 +94,12 @@ func (c *Config) resolveNetworks(flagSet *pflag.FlagSet) (network.Type, error) {
 	c.Ethereum.Network = clientNetwork.Ethereum()
 	c.Bitcoin.Network = clientNetwork.Bitcoin()
 
+	logger.Infof(
+		"using [%v] Ethereum network and [%v] Bitcoin network",
+		c.Ethereum.Network,
+		c.Bitcoin.Network,
+	)
```

Comment:
This check is based on the flags declared by the user in config. Could we add two checks?
1. If Ethereum is testnet, then Bitcoin also has to be testnet, If Ethereum is mainneg then Bitcoin also has to be mainnet.
2. That the actual Ethereum and Electrum servers the node is connected to supports the desired networks?
We have a similar solution in the Electrum TS library: https://github.com/keep-network/tbtc-v2/blob/7f48d6362f8bfc9a2931c4087e5eba28efee5370/typescript/src/electrum.ts#L208-L224

Reply:
This doesn't have to be handled in this PR, as the scope of this PR was significantly reduced to just printing general information.

Reply:
Can you please open an issue to track this work?

Reply:
https://github.com/keep-network/keep-core/issues/3602